virt/vm_networking/virt-dedicated-network-live-migration.adoc

```diff
@@ -0,0 +1,18 @@
+:_content-type: ASSEMBLY
+[id="virt-dedicated-network-live-migration"]
+= Configuring a dedicated network for live migration
+include::_attributes/common-attributes.adoc[]
+:context: virt-dedicated-network-live-migration
+toc::[]
+
+You can configure a dedicated xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-connecting-vm-to-linux-bridge[Multus network] for live migration. A dedicated network minimizes the effects of network saturation on tenant workloads during live migration.
+
+include::modules/virt-configuring-secondary-network-vm-live-migration.adoc[leveloffset=+1]
+
+include::modules/virt-selecting-migration-network-ui.adoc[leveloffset=+1]
+
+[id="additional-resources_virt-migrating-vm-on-secondary-network"]
+[role="_additional-resources"]
+== Additional resources
```

Comment:
I am not sure if the order matters, but our examples show `role` before `id`

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings